### PR TITLE
Organization bug bash: Manage Packages page

### PR DESCRIPTION
**Pull request by chenriksson**
Fixes the following bug bash issues:
- #5299 comma separators for package and download counts
- #5304 flash of empty containers
- #5310 Layout issues (except binding filter to namespaces / requests will be separate PR)
- #5298 Org icon too high
- #5306 Owner link whitespace fix
- #5307 Visual separation between owners

Page Screenshot:

image

Package row w/ owners screenshot:

image

### CODE REVIEW

File: src/NuGetGallery/Views/Users/Packages.cshtml [hunk 1]
```diff
@@ -4,92 +4,82 @@
...
-                    @ViewHelpers.Section(this,
-                        "requests-incoming",
-                        "Incoming",
-                        @<text>
-                            @Html.Partial("_OwnerRequestsList", Model.OwnerRequests.Incoming)
-                        </text>)
-                }
+            @if (Model.OwnerRequests.Incoming.RequestItems.Any())
+            {
+                @ViewHelpers.Section(this, "requests-incoming",
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/Packages.cshtml]: You could probably pretty easily make a helper function to reuse for both `Incoming` and `Outgoing`.

```
@helper OwnerRequestsSection(OwnerRequestListModel model, string name)
{
    @ViewHelpers.Section(this, "requests-" + name",
        @<text>Ownership Requests (@name)</text>,
        @<text>@Html.Raw(model.RequestItems.Count() + " request" + (ownerRequestsIncomingCount == 1 ? "" : "s"))</text>,
        ...
}
```

You could potentially add this into `_OwnerRequestsList` itself as well.

File: src/NuGetGallery/Views/Users/Packages.cshtml [hunk 2]
```diff
@@ -4,92 +4,80 @@
     ViewBag.Title = "Manage My Package";
     ViewBag.Tab = "Packages";
     Layout = "~/Views/Shared/Gallery/Layout.cshtml";
+    
+    var namespacesCount = Model.ReservedNamespaces.ReservedNamespaces.Count();
+    var ownerRequestsIncomingCount = Model.OwnerRequests.Incoming.RequestItems.Count();
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/Packages.cshtml]: I really like the change from "Incoming" to "Received" and from "Outgoing" to "Sent", but ideally we should update the field names on the class to match this naming as well.

Comment (chenriksson) [reply src/NuGetGallery/Views/Users/Packages.cshtml]: Fixed

File: src/NuGetGallery/Views/Users/Packages.cshtml (hunk 1 above)
Comment (chenriksson) [reply src/NuGetGallery/Views/Users/Packages.cshtml]: Will consider in next PR. For #5310 I need to update the namespaces and requests sections to also include organizations and support the account filter at the top.

File: src/NuGetGallery/Views/Users/Packages.cshtml [hunk 3]
```diff
@@ -114,14 +102,12 @@
...
-                                <a data-bind="attr: { href: ProfileUrl }">
-                                    <span data-bind="text: Username"></span>
-                                </a>
+                                <i class="ms-Icon ms-Icon--Group" aria-hidden="true" data-bind="visible: IsOrganization" style="vertical-align: -2px"></i>
+                                <a data-bind="attr: { href: ProfileUrl }"><span data-bind="text: Username"></span></a><!-- ko if: ($index() < ($parent.Owners.length - 1)) -->,&nbsp;<!-- /ko -->
                             </span>
                         </td>
                         <td class="align-middle text-nowrap">
-                            <span data-bind="text: DownloadCount"></span>
+                            <span data-bind="text: ko.unwrap(DownloadCount).toLocaleString()"></span>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/Packages.cshtml]: Seems like this should be a FormattedDownloadCount computed

**Comment by joelverhagen** [comment]
The slash between the package count and the download count seems weird. A comma seems like a better fit.

File: src/NuGetGallery/Views/Users/Packages.cshtml [hunk 4]
```diff
@@ -114,14 +102,12 @@
                         </td>
                         <td class="align-middle">
                             <span class="ms-noWrap" data-bind="foreach: Owners">
-                                <i class="ms-Icon ms-Icon--Group" aria-hidden="true" data-bind="visible: IsOrganization"></i>
-                                <a data-bind="attr: { href: ProfileUrl }">
-                                    <span data-bind="text: Username"></span>
-                                </a>
+                                <i class="ms-Icon ms-Icon--Group" aria-hidden="true" data-bind="visible: IsOrganization" style="vertical-align: -2px"></i>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/Packages.cshtml]: Try to avoid inline css. Other icons are doing this.

**Comment by chenriksson** [comment]
> The slash between the package count and the download count seems weird. A comma seems like a better fit.

@joelverhagen Missed this, but will incorporate in a follow-up PR

**Pull request merged**